src/mongocxx/hint.hpp

```diff
@@ -155,10 +157,6 @@ MONGOCXX_INLINE hint::operator bsoncxx::types::value() const {
     return to_value();
 }
 
-MONGOCXX_INLINE hint::operator bsoncxx::document::value() const {
```

Comment:
You de-inlined the definition of this function, but you didn't remove the `MONGOCXX_INLINE` from its declaration.